packages/ui-library/src/components/textarea/index.css.ts

```diff
@@ -34,11 +34,15 @@ export const staticStyles = typeSafeNestedCss/*css*/ `
   .blr-textarea-info-container {
     display: flex;
     justify-content: right;
-
-    &.hint, &.error {
-      justify-content: space-between;
-    }
   }
+  
+  .blr-textarea-info-container.error {
```

Comment:
For consistency, please follow the similar pattern like other:
.blr-textarea-info-container {
&.error {...}
&.hit {...}
}